### PR TITLE
Fix indentation of lambdas during declaration

### LINKED ISSUE #236: Field with lambda
I'm not sure how this should format, but I think this could be improved.
```c#
    private static readonly Func<
        SyntaxTrivia,
        bool
    > s_isVisualBasicCommentTrivia =
        (syntaxTrivia) => syntaxTrivia.IsKind(
            VisualBasic.SyntaxKind.CommentTrivia
        );
```

### LINKED ISSUE #246: Extra Indentation with Lambda
0.9.5 introduced this probably as part of https://github.com/belav/csharpier/pull/240
```c#
Action find = () => EntryPointDiscoverer.FindStaticEntryMethod(
        typeof(IEnumerable<>).Assembly
    );
// should be
Action find = () => EntryPointDiscoverer.FindStaticEntryMethod(
    typeof(IEnumerable<>).Assembly
);
```
#238 may be related

### PR DESCRIPTION
**Pull request by divyanshshukla13**
Closes #236 
Closes #246

### CODE REVIEW

**Review by shocklateboy92 — CHANGES_REQUESTED** [review]
Could you please update add a test that covers this change?
It should be fairly straight-forward - you just need to create a `.cst` file in `Src/CSharpier.Tests/TestFiles/VariableDeclaration/` and reference it in `_VariableDeclarationTests.cs`.

**Comment by belav** [comment]
> It should be fairly straight-forward - you just need to create a `.cst` file in `Src/CSharpier.Tests/TestFiles/VariableDeclaration/` and reference it in `_VariableDeclarationTests.cs`.
The standard for tests is a single file within each directory, unless there are special cases that warrant being in their own file. And this bug shows up with a FieldDeclaration, which would mean updating `/TestFiles/FieldDeclaration/FieldDeclarations.cst`, although it may also present itself when declaring a variable as well.

File: Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst [hunk 1]
```diff
@@ -51,6 +51,10 @@ public class ClassName
         SomeLongTypeName,
         SomeOtherLongTypeName_______________________________
     >();
+
+    private static readonly Func<SyntaxTrivia, bool> s_isVisualBasicCommentTrivia = (
```
Comment (shocklateboy92) [inline Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst:55]: Do people really use Hungarian notation (e.g. `s_`) in C#? :open_mouth:

**Review by shocklateboy92 — APPROVED** [review]
Thanks for your contribution to CSharpier!
Feel free to drop by on our [Discord channel](https://discord.gg/HfAKGEZQcX) sometime

File: Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst (hunk 1 above)
Comment (divyanshshukla13) [reply Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst:55]: Not really. I just picked up this example from the linked issue.

Comment (belav) [reply Src/CSharpier.Tests/TestFiles/FieldDeclaration/FieldDeclarations.cst:55]: I've definitely run into it, which is why it ended up in the linked issue.

**Pull request merged**